grafana/configmaps.libsonnet

```
@@ -144,5 +146,8 @@ local configMap = k.core.v1.configMap;
         for folder in std.objectFields($.grafanaDashboardFolders)
       ]);
 
-    k.util.volumeMounts(mounts),
+    k.util.volumeMounts(mounts)
+    + deployment.mixin.spec.template.metadata.withAnnotationsMixin({
+      'dashboards-hash': std.md5(std.toString($.grafanaDashboardFolders)),
```

```suggestion
      'grafana-dashboards-hash': std.md5(std.toString($.grafanaDashboardFolders)),
```

For consistency with the other annotations, what do you think about ☝️?